server/router/server.go

```diff
@@ -154,7 +154,6 @@ func NewRouter(_ context.Context, h models.HandlerInterface, port int, g http.Ha
 	gMux.Handle("/api/meshmodels/categories", h.ProviderMiddleware(h.AuthMiddleware(http.HandlerFunc(h.GetMeshmodelCategories), models.NoAuth))).Methods("GET")
 	gMux.Handle("/api/meshmodels/models", h.ProviderMiddleware(h.AuthMiddleware(http.HandlerFunc(h.GetMeshmodelModels), models.NoAuth))).Methods("GET")
 	gMux.Handle("/api/meshmodels/models/{model}", h.ProviderMiddleware(h.AuthMiddleware(http.HandlerFunc(h.GetMeshmodelModelsByName), models.NoAuth))).Methods("GET")
-	gMux.Handle("/api/meshmodels/components", h.ProviderMiddleware(h.AuthMiddleware(http.HandlerFunc(h.GetAllMeshmodelRelationships), models.NoAuth))).Methods("GET")
```

Comment:
@abdullah1308 Is this intentional?

Reply:
@MUzairS15 yes as the following endpoint exists causing the above line to do nothing -

https://github.com/meshery/meshery/blob/051d838aadc60edcd6bed1bf9f0271b3fd07b126/server/router/server.go#L155

This endpoint has the exact same handlers and is functional -

https://github.com/meshery/meshery/blob/051d838aadc60edcd6bed1bf9f0271b3fd07b126/server/router/server.go#L172